### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'sign' in ncvx

### ISSUE THREAD

**Issue by Zeinab-E** [cvxpy/cvxpy#1486]
Hi,

I got the following error when running the simple example of ncvx (https://github.com/cvxgrp/ncvx), I installed cvxpy=1.1.3.
----------------------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-38-3e4acb329b39> in <module>()
     16 objective = cp.sum_squares(A @ x - b)
     17 prob = cp.Problem(cp.Minimize(objective), [])
---> 18 prob.solve(method="NC-ADMM")
     19 print(f"Objective = {objective.value}")
     20 print(f"x = \n{x.value}")

3 frames
/usr/lib/python3.7/multiprocessing/pool.py in get(self, timeout)
    655             return self._value
    656         else:
--> 657             raise self._value
    658 
    659     def _set(self, i, obj):

TypeError: __init__() got an unexpected keyword argument 'sign'
--------------------------------------------------------------------------------
I also change the Parameter(shape, nonneg=True) instead of Parameter(shape, sign='positive'). but it still is not working and I got the error about "sign".

I would greatly appreciate it if you could help me.

**Comment by SteveDiamond** [comment]
You should post this as an NCVX issue. What version of NCVX are you using? Can you try installing NCVX from source?

**Comment by Zeinab-E** [comment]
Sorry again, 
For installing ncvx=0.2.0 , it needs scsprox>=0.2. But there is just two versions of (0.1.0a1 and 0.1.0a2). So, the requirements for ncvx=0.2.0 are not satisfied and can't install from the source.

ERROR: Could not find a version that satisfies the requirement scsprox>=0.2 (from ncvx) (from versions: 0.1.0a1, 0.1.0a2)

**Comment by SteveDiamond** [comment]
Please follow the installation instructions: https://github.com/cvxgrp/ncvx#installation

**Comment by Zeinab-E** [comment]
I really followed the installation instruction but it is not working for installing ncvx=0.2.0 since it requires scsprox>=0.2 and it is not available.
As you have written in setup.py :  install_requires=["cvxpy >= 1.1", "lap", "scsprox >= 0.2", "numpy"],

I would greatly appreciate it if you could check it.

**Comment by SteveDiamond** [comment]
You installed scsprox from source?

**Comment by Zeinab-E** [comment]
Yes, I installed it from:  [https://github.com/bettbra/scsprox](url) as you mentioned.  I installed scsprox from the source by "pip setup.py install".
After installation, I checked the version by "pip list". The version is scsprox 0.1.0a2. Actually, I did not find the scsprox=0.2 as it is required for installing ncvx=0.2.0. 
![2](https://user-images.githubusercontent.com/82809946/135331598-7b00350e-1273-4b38-899c-28149bc24763.png)

**Comment by SteveDiamond** [comment]
To install from source it's ``python setup.py install``.

**Comment by SteveDiamond** [comment]
Also be sure to uninstall scsprox fully before you install it from source.

**Comment by Zeinab-E** [comment]
> To install from source it's `python setup.py install`.

Sorry, I wrote it wrongly. yes I installed it by "python setup.py install"
I will check it out by reinstalling it again. Thanks

**Comment by Transurgeon** [comment]
closing this issue as it was related to ``ncvx`` and not ``cvxpy``.
